hushline/model/field_definition.py

```diff
@@ -20,16 +20,20 @@ class FieldDefinition(Model):
     __tablename__ = "field_definitions"
     __table_args__ = (UniqueConstraint("username_id", "sort_order"),)
 
-    id: Mapped[int] = mapped_column(primary_key=True)
-    username_id: Mapped[int] = mapped_column(db.ForeignKey("usernames.id"), index=True)
+    id: Mapped[int] = mapped_column(primary_key=True, nullable=False)
+    username_id: Mapped[int] = mapped_column(
+        db.ForeignKey("usernames.id"), nullable=True, index=True
+    )
     username: Mapped["Username"] = relationship(back_populates="message_fields")
-    label: Mapped[str] = mapped_column(db.String(255))
-    field_type: Mapped[FieldType] = mapped_column(SQLAlchemyEnum(FieldType))
-    required: Mapped[bool] = mapped_column()
-    enabled: Mapped[bool] = mapped_column()
-    encrypted: Mapped[bool] = mapped_column()
-    choices: Mapped[list[str]] = mapped_column(type_=JSONB)
-    sort_order: Mapped[int] = mapped_column()
+    label: Mapped[str] = mapped_column(db.String(255), nullable=False)
+    field_type: Mapped[FieldType] = mapped_column(
```

Comment:
This might get things in alignment, but it seems wrong to allow these to be nullable at all. I guess this is something to open a followup ticket for.

Reply:
Yeah that does seem wrong. It would probably be fine to just change nullable to False and make a migration, but since I'm not entirely sure and don't want to break anything, let's do that in a future PR